src/main/java/org/manifold/compiler/front/FrontendBuildException.java

```diff
@@ -0,0 +1,12 @@
+package org.manifold.compiler.front;
+
+/**
+ * Use this for errors that should be reported directly to the user.
+ * Non-internal errors should be translated to a FrontendBuildException somewhere in the pipeline.
+ */
+public class FrontendBuildException extends Error {
```

Comment:
The person who made that decision is then one that left.  I don't see any reason not to switch to extending `RuntimeException`